_posts/en/newsletters/2023-05-24-newsletter.md

```diff
@@ -0,0 +1,175 @@
+    effectively the case if merges are common.
+
+    That means a protocol which can compress the state of Bitcoin can
+    also be adapted to compress the state of a token's history, even one
+    where merges are common.  The authors describe how they would
+    accomplish that.  Their goal is to produce a proof that each
+    previous transfer of the token followed the token's rules, including
+    using their proofs for Bitcoin to prove that each previous transfer
+    was anchored in the block chain.  Alice could then transfer the
+    tokens to Bob and give him a short constant-sized proof of validity;
+    Bob could verify the proof to know that the transfer happened at a
+    certain block height and paid his token wallet, giving him exclusive
+    control over the tokens.
+
+    Although the paper frequently mentions additional work that needs to
+    be performed in order to make both protocols usable in production,
+    we find this to be encouraging progress towards a
+    [feature][coinwitness] that Bitcoin developers have desired for over
+    a decade.
+
+## Waiting for confirmation #2: FIXME:glozow title
+
+_A limited weekly series about transaction relay, mempool inclusion, and
+mining transaction selection---including why Bitcoin Core has a more
+restrictive policy than allowed by consensus and how wallets can use
+that policy most effectively._
+
+{% include specials/policy/en/02-cache-utility.md %}
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Core Lightning 23.05rc2][] is a release candidate for the next
+  version of this LN implementation.
+
+- [Bitcoin Core 23.2rc1][] is a release candidate for a maintenance
+  release of the previous major version of Bitcoin Core.
+
+- [Bitcoin Core 24.1rc3][] is a release candidate for a maintenance
+  release of the current version of Bitcoin Core.
+
+- [Bitcoin Core 25.0rc2][] is a release candidate for the next major
+  version of Bitcoin Core.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27021][] Implement Mini version of BlockAssembler to calculate mining scores FIXME:harding (hoping glozow or Xekyo will volunteer, though)
```

Comment:
If neither of them is available, I could help out. (I don't know this PR as well as they do, but I know it fairly well.)

Reply:
I can give it a spin.